template/src/index.js

```diff
@@ -3,16 +3,24 @@ import 'regenerator-runtime/runtime'
 import {registerHtml, start} from 'tram-one'
 import ColorHeader from './components/ColorHeader'
 import './styles.css'
+import { useColor } from './hooks/ColorHook'
 
 const html = registerHtml({
 	ColorHeader
 })
 
 const home = () => {
+  const [color, incrementColor] = useColor()
 	return html`
     <div>
       <ColorHeader />
       Thank you for using Tram-One!
+      <div
+        style="color:${color}"
+        onclick=${incrementColor}
+      >
+       The color is controlled by global state 
+      </div>
```

Comment:
This is great! If we just want to wrap the `Thank you for using Tram-One!` in a div with color, I think that'll be fine. We can just use the following in that case
```
const [color] = useColor()

...

<div style="color:${color}"> Thank you for using Tram-One! </div>

```
```